test/facility-locator/00-required.e2e.spec.js

```diff
@@ -18,11 +20,16 @@ module.exports = E2eHelpers.createE2eTest(
     .setValue('input[name="streetCityStateZip"]', 'Seattle, WA');
 
     client
-      .axeCheck('.main')
       .click('input[type="submit"]');
 
     client
       .axeCheck('.main');
```

Comment:
Probably could just remove the `client` in this block and just append the `axeCheck` to the chain above, right?